viewing_party/main.py

```diff
@@ -0,0 +1,227 @@
+#************** WAVE 1 ***************
+
+def create_movie(title, genre, rating):
+    
+    new_movie = {}
+    
+    if type(title) == str and type(genre) == str and type(rating) == float:
+        new_movie["title"] = title 
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+
+        return new_movie
+
+    else: 
```

Comment:
Style: You should avoid blank lines in front of an `else` or `elif`.  

(An `else` or `elif` can't exist independent of an `if` so we try to "connect" them visually.)